src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -1417,23 +1434,24 @@ void GNAGraphCompiler::EltwisePrimitive(InferenceEngine::CNNLayerPtr layer) {
     void* ptr_biases = nullptr;
 
     auto& currentComponent = dnnComponents.addComponent(layer->name, "diagonal");
-    dnn->InitAffineComponent(
-        currentComponent,
-        num_rows_in + num_padding,
-        num_columns_in,
-        num_rows_out + num_padding,
-        inputs2Bytes->getPrecision().size(),
-        outputs->getPrecision().size(),
-        // TODO: only fp32 and Int16 tested
-        quantized == nullptr ? inputs2Bytes->getPrecision().size() : (gna_config.gnaFlags.input_low_precision ? 1 : 2),
-        quantized == nullptr ? inputs4Bytes->getPrecision().size() : (gna_config.gnaFlags.input_low_precision ? 1 : 4),
-        GetScaleFactor(layer, QuantizedDataType::weights),
-        GetScaleFactor(layer, QuantizedDataType::output),
-        ptr_inputs,
-        ptr_outputs,
-        ptr_weights,
-        ptr_biases,
-        true);
+    dnn->InitAffineComponent(currentComponent,
```

Comment:
was this not formatted properly earlier?

Reply:
I think formatting changed due to additional casting of the arguments